gnome-initial-setup/eos-demo-mode-force-gdm-restart

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+systemctl reload gdm.service
```

Comment:
So I gave this a try, but I'm getting a strange error when calling the DBus method - it just says that `org.freedesktop.systemd1` exited with status 1. Nothing else in the journal that gave me any indication of what was going on. Any ideas?